### PR TITLE
Fix compat with Symfony 5

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,14 +12,15 @@
             "homepage": "https://dunglas.fr"
         }
     ],
+    "minimum-stability": "dev",
```
Comment (soyuka) [inline composer.json]: should we wait until symfony's released before merging this?

Comment (dunglas) [reply composer.json]: Just drop this line, dev versions are already tested in a dedicated CI build.

Comment (teohhanhui) [reply composer.json]: And revert all the `--prefer-stable`

Comment (teohhanhui) [reply composer.json]: But this PR cannot be merged until Symfony 4.3.7 is released.

**Comment by fabpot** [comment]
4.3.7 has been released now :)

**Comment by teohhanhui** [comment]
CI failure is I/O error: https://github.com/api-platform/core/pull/3242/checks?check_run_id=298039241

Looks like it just needs another force push, since GitHub Actions doesn't support re-runs yet.

**Comment by dunglas** [comment]
Or just this 😉. Thanks @nicolas-grekas!

**Pull request merged**